dotcom-rendering/src/components/Elements.amp.tsx

```diff
@@ -97,6 +97,16 @@ export const isAmpSupported = ({
 			return false;
 	}
 
+	if (
+		elements.find(
```

Comment:
[`Array.prototype.some()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) is probably more semantic?

```suggestion
		elements.some(
```